packages/@o3r/design/src/core/design-token/design-token-specification.interface.ts

```diff
@@ -0,0 +1,252 @@
+/** Metadata information added in the design token extension for Metadata extraction */
+export interface DesignTokenMetadata {
+  tags?: string[];
+  /** Description of the variable */
+  label?: string;
+  /** Name of a group of variables */
+  category?: string;
+}
+
+/** Design Token Group Extension fields supported by the default renderer */
+export interface DesignTokenGroupExtensions {
+  /** Indicate the file where to generate the token */
+  o3rTargetFile?: string;
+  /**
+   * Indicate that the variable does not need to be generated (up to the generator or its options).
+   * If not generated, the generator will use its targeted value in the variable referring to it.
```

Comment:
I don't know if this is clearer: 

Indicate that the variable does not need to be generated. 
It is up to the generator to describe how to render private variables. It can choose to ignore the private extension, it can provide a dedicated renderer (for exemple to prefix it with '_') or it can decide to skip the generation straight to its referenced value.